covidWallet/app/components/SecurityScreen.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { View,Image,Text,StyleSheet} from 'react-native';
+import { useState } from 'react';
+import PrimaryButton from '../components/PrimaryButton';
+import { PRIMARY_COLOR } from '../constants/constants';
+const img = require('../assets/images/face.jpg');
+function SecurityScreen({navigation}) {
+
+  nextHandler = () => {     
+        navigation.navigate('NotfiyMeScreen');
+    
+}
+
+  return (
+    
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+     <Image style={{ width:280, height:240  }}
+              source={img}
+    />
+    <View>
+        <Text style={styles.TextContainer}>Enable your Face ID for security purpose.</Text>
```

Comment:
Missing sub-text:

'Using biometric and passcode security significantly reduces the chances of your account will be compromised in case your phone is stolen or lost.'